js/plot/basal.js

```diff
@@ -182,9 +182,9 @@ module.exports = function(pool, opts) {
             .text(function(d) {
               /* jshint laxbreak: true */
               return isAutomated
-                ? _.get(AUTOMATED_BASAL_LABELS, source, 'A').charAt(0)
-                : _.get(SCHEDULED_BASAL_LABELS, source, 'M').charAt(0);
-            });
+                ? t('A_Label').charAt(0)
+                : t('M_Label').charAt(0);
```

Comment:
Since theses labels are now translated, why keep the substring (`charAt()`) ? 
```suggestion
return isAutomated ? t('A_Label') : t('M_Label');
```

Reply:
As we have to display only the first letter of the label, I prefer to keep it that way so that we don't have to check that the translations are correct and contain only 1 letter.
![image](https://user-images.githubusercontent.com/22294614/63586177-0d984400-c5a1-11e9-97c4-55ec09cc6b04.png)